lib/parser.js

```diff
@@ -1450,6 +1453,14 @@ function parseStatement(parseContext) {
     return handleMaker(parseContext);
   }
 
+  if (tokens[0] === 'git') {
+    return classHandlers.handleGit(parseContext);
+  }
+
+  if (tokens[0] === 'sit' ) {
```

Comment:
whitespace?

Reply:
Ah, got it, the others are `if(x === 'foo')`. I should probably get a linter on this....